main.css

```diff
@@ -11,9 +11,15 @@
     font-weight: 600;
 }
 
+.module.MMM-MonthCalendar .module-header {
+    text-align: center;
+}
+
 .module.MMM-MonthCalendar .calendar-daysinweek div,
 .module.MMM-MonthCalendar .calendar-days div {
     padding: 4px 0;
+    margin-left: 8px;
+    margin-right: 8px;
```

Comment:
I prefer to write `margin: 0 8px`. Looks a bit smoother IMHO 😉

Reply:
Looks indeed better :)